src/components/skeleton/Skeleton.vue

```diff
@@ -55,8 +55,9 @@ export default {
             }))
         }
         return createElement('div', {
+            ...context.data,
             staticClass: 'b-skeleton',
-            class: [ context.props.size, context.props.position, { 'is-animated': context.props.animated } ]
+            class: [ context.data.attrs.class, context.props.size, context.props.position, { 'is-animated': context.props.animated } ]
```

Comment:
Are you sur is it ok? I usually use .`staticClass` and not .class

Reply:
Do you know the proper way to test this ? 

If I [do ](https://github.com/buefy/buefy/pull/3642/files#diff-6c1222579876270e0ad765a3dc4efc91824ab4e1395b805b74d352d15c2b843bR8-R12):
```js
wrapper = mount(BSkeleton, {
  attrs: {
    class: 'custom-class'
  }
})
```
I don't have `staticClass` set.

Reply:
I mean in the core code, you should use `context.data.attrs.class` instead of `context.data.attrs.class`